src/wwwroot/locales/en-translation.js

```diff
@@ -282,6 +282,7 @@
   "attr_api": "API & SMTP connection",
   "attr_dkim": "DKIM Settings",
   "attr_metrics_and_reports": "Metrics and reports",
+  "attr_dedicated_ip": "Dedicated IP",
```

Comment:
In order to avoid issues in the future, what do you think about showing `Dedicated IPs: 1` in place of `Dedicated IP`?